airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -381,175 +381,302 @@ static Optional<ContainerOrchestratorConfig> getContainerOrchestratorConfig(fina
     }
   }
 
-  private static void launchWorkerApp(final Configs configs, final DSLContext configsDslContext, final DSLContext jobsDslContext) throws IOException {
-    MetricClientFactory.initialize(MetricEmittingApps.WORKER);
+  private static AirbyteApiClient getApiClient(final Configs configs) {
+    final var authHeader = configs.getAirbyteApiAuthHeaderName();
+
+    // control plane workers communicate with the Airbyte API within their internal network, so https
+    // isn't needed
+    final var scheme = configs.isControlPlaneWorker() ? "http" : "https";
```

Comment:
Is this always true, even on K8s?

Reply:
Continued in child PR here: https://github.com/airbytehq/airbyte/pull/15904/files#r953197905